### PR TITLE
fix image issue with public member photo

### PR DESCRIPTION
**Pull request by brettski**

### CODE REVIEW

**Comment by csell5** [comment]
#### 📝Changed routes:


| [**/member/\[memberSlug\]**](https://that-website-git-brettski-fix465-no-public-member-photo.that-conference.now.sh/member/[memberSlug]) |
|:-:|
|<a href="https://that-website-git-brettski-fix465-no-public-member-photo.that-conference.now.sh/member/[memberSlug]">Screenshot of /member/[memberSlug]</a><br /><sup><a href="https://images.weserv.nl?url=https://microlink-cdn.s3.us-east-1.amazonaws.com/pspft097sDTl7MPQoz31NnCdWjc_.png">(view full size)</a> |







Commit d4d34b605479e0a4fea7a9626a06e6b66639c340 (https://that-website-git-brettski-fix465-no-public-member-photo.that-conference.now.sh).

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "node": "^10"
   },
   "scripts": {
-    "dev": "next ./ -p 3001",
+    "dev": "next ./ -p 3000",
```
Comment (csell5) [inline package.json]: that was on 3001 as `now dev` was on 3000

Comment (brettski) [reply package.json]: Would the be run at the same time? I figured keep the ports the same for cookies, etc.

Comment (csell5) [reply package.json]: prolly not. I've done it a few times to compare a production output vs the now dev run.  but cookies is also a good reason.

**Pull request merged**